### ISSUE TITLE
Unable to use fs in filter

### ISSUE THREAD

**Issue by Orbifoldt** [rangav/thunder-client-support#949]
**Describe the bug**

Unable to use the `fs` module in a custom filter. 

**To Reproduce**

I took the example filter file and tried loading `fs` like so:
```javascript
async function customFilter(input) {
    console.log("Loading node module - moment");

    var fs = await tc.loadModule("fs");
    console.log(fs === undefined);
  
    var moment = await tc.loadModule("moment");
    console.log(moment === undefined)
}
```

In the log it shows:
```
---------------- Running Request: ----------------
Environment Used: Development (Active)
Script Log: Loading node module - moment 
Script Log: Error loading node package: fs - Cannot find c:\Users\<My-user>\.vscode\extensions\rangav.vscode-thunder-client-2.2.4\dist\node_modules\fs\index.js in plugin fs
Script Log: true
Script Log: false
```
This shows that it can load `moment` fine, but for `fs` it just tries downloading: https://www.npmjs.com/package/fs which isn't correct.

I am no javascript expert, but I think that `fs` is a special kind of package. So, I did also try adding a require at the top of the file:
```
const fs = require("fs");
```
But that doesn't work either, when the function runs `fs` is still undefined.

**Expected behavior**

To be able to load `fs` either using `require` or `tc.loadModule`

**Platform:** 
 - OS: Win 10
 - vscode version: 1.72.2
 - extension version: 2.2.4

**Your Team Size Using TC:**
1

**Comment by rangav** [comment]
@Orbifoldt thanks for the feedback 

`fs` is not enabled, can you let us know why you like to use fs module 

just trying to understand use-case

**Comment by Orbifoldt** [comment]
@rangav , I want to test a file upload using a rest service, but this service requires the files to be send as a base64 encoded string in the json body. So my goal is to read a filepath from a variable, then open that file, encode it and then return it as a string.

So I'd like to create a request body like so:
```json
{
  "processId": 123
  "file": "{{myTestFilePath | encodeFileAsBase64}}"
}
```
And that method would just call something like `fs.readFileSync(filePath, {encoding: 'base64'})`

**Comment by rangav** [comment]
Thanks for sharing your usecase

will enable fs in the next update.

also will think if they is any simpler way to do this.

**Comment by Orbifoldt** [comment]
Thanks!

Just thinking aloud: perhaps adding an `@` symbol could expand a path as file content, sort of how it works in curl? 
Then you could do something like `{{@myTestFilePath | btoa}}` 

Or a separate readFile filter? `{{myTestFilePath | readFile | btoa}}`

**Comment by rangav** [comment]
> Or a separate readFile filter? `{{myTestFilePath | readFile | btoa}}`

this syntax is better, thanks for suggestion

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.3.0`

Please use `{{myTestFilePath | readFile | btoa}}`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.3.0

Please let me know your feedback.
